### PR TITLE
build 1.2.3

### PR DESCRIPTION
**Pull request by boldorider4**
snowflake-ml-python 1.2.3

**Destination channel:** Defaults

### Links

- [PKG-4164]
- [Upstream repository](https://github.com/snowflakedb/snowpark-python/)
- [Upstream changelog/diff](https://github.com/snowflakedb/snowflake-ml-python/compare/1.2.2...1.2.3)

### Notes
- dependencies have been reordered to match order in [requirements.yml](https://github.com/snowflakedb/snowflake-ml-python/blob/1.2.3/requirements.yml)

[PKG-4164]: https://anaconda.atlassian.net/browse/PKG-4164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/snowflakedb/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 0d692c13eb2f1b8f6ecf0bf73bbb79915dadbd7913b4f8ae1eb6ab798f86e860
+  sha256: ed68d0c2b9b5b8c7b57a13ae5277d03aee09d20d589aaeb630fae2f3bdddaca4
 
 build:
   number: 0
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:13]: They now support python 3.11, see
* https://github.com/snowflakedb/snowflake-ml-python/compare/1.2.2..1.2.3#diff-3d3007f9af2e340808318a19ea9afbe63e8a93753bf4e0696ad44e9ecb11b8ecR25
* https://github.com/snowflakedb/snowflake-ml-python/compare/1.2.2..1.2.3#diff-544556920c45b42cbfe40159b082ce8af6bd929e492d076769226265f215832fR28
* https://inspector.pypi.io/project/snowflake-ml-python/1.2.3/packages/52/54/7ae7bc8c18358bd5dd9cadcd325da8ad0e3627a7e94bb6d255661beab0b5/snowflake_ml_python-1.2.3-py3-none-any.whl/snowflake_ml_python-1.2.3.dist-info/METADATA#line.233

File: recipe/meta.yaml [hunk 2]
```diff
@@ -55,11 +54,13 @@ requirements:
     - typing-extensions >=4.1.0,<5
     - xgboost >=1.7.3,<2
     - cachetools >=3.1.1,<5
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:56]:
```suggestion
    - cachetools >=3.1.1,<6
```
See https://inspector.pypi.io/project/snowflake-ml-python/1.2.3/packages/52/54/7ae7bc8c18358bd5dd9cadcd325da8ad0e3627a7e94bb6d255661beab0b5/snowflake_ml_python-1.2.3-py3-none-any.whl/snowflake_ml_python-1.2.3.dist-info/METADATA#line.238

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml:13]: Thank you!

Comment (boldorider4) [reply recipe/meta.yaml:13]: oh, actually only from `py312` onward the build is skipped.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by ViridianMelody — APPROVED** [review]

**Pull request merged**